pkg/asset/imagebased/image/ignition.go

```diff
@@ -39,6 +42,7 @@ func (i *Ignition) Dependencies() []asset.Asset {
 		&ImageBasedInstallationConfig{},
 		&RegistriesConf{},
 		&PostDeployment{},
+		&configimage.InstallConfig{},
```

Comment:
Does this mean that in order to create the IBI installation ISO we now require the user to prepare an `install-config.yaml` ? 

If yes, then I think  we need to put whatever fields we need from the `install-config.yaml` into the `imagebased-installation-config.yaml`.  

I understand that it's not super clear by looking into the respective [design doc](https://github.com/openshift/enhancements/blob/master/enhancements/installer/image-based-installer.md) (or just the codebase), but having an `install-config.yaml` as input for both the create installation ISO and create config ISO subcommands leads to a confusing user experience and we [decided](https://github.com/openshift/enhancements/pull/1617#discussion_r1654313325) that it fits only to the create config ISO subcommand.

Reply:
Just to clarify what I wrote above, we do have the `networkConfig` field in the [IBI installation config](https://github.com/openshift/installer/blob/3474a3e90f7840818b616994539c5d6b79365f62/pkg/types/imagebased/imagebased_config_types.go#L102) so I think we should fetch it from there, no need for the `install-config.yaml`.

Please let me know if you think I'm missing anything.